src/MICore/MICoreResources.resx

```diff
@@ -204,4 +204,7 @@ Error: {1}</value>
   <data name="Status_BreakpointPending" xml:space="preserve">
     <value>Module containing this breakpoint has not yet loaded or the breakpoint address could not be obtained.</value>
   </data>
+  <data name="Error_InvalidLocalCoreDumpPath" xml:space="preserve">
+    <value>Invalid path to core dump file path '{0}'. File must be a valid file name that exists on the Visual Studio computer.</value>
```

Comment:
I copied it from the Error_InvalidLocalExePath resource string and modified it. Other strings in this resx file have references to Visual Studio. I'll log a bug to remove those references.